path_static_roles_test.go

```diff
@@ -489,38 +364,14 @@ func TestRoles(t *testing.T) {
 		b, storage := getBackend(false)
 		defer b.Cleanup(context.Background())
 
-		data := map[string]interface{}{
-			"binddn":      "tester",
-			"bindpass":    "pa$$w0rd",
-			"url":         "ldap://138.91.247.105",
-			"certificate": validCertificate,
-		}
-
-		req := &logical.Request{
-			Operation: logical.CreateOperation,
-			Path:      configPath,
-			Storage:   storage,
-			Data:      data,
-		}
-
-		resp, err := b.HandleRequest(context.Background(), req)
-		if err != nil || (resp != nil && resp.IsError()) {
-			t.Fatalf("err:%s resp:%#v\n", err, resp)
-		}
+		configureOpenLDAPMount(t, b, storage)
 
-		data = map[string]interface{}{
+		data := map[string]interface{}{
 			"username": "hashicorp",
 			"dn":       "uid=hashicorp,ou=users,dc=hashicorp,dc=com",
 		}
 
-		req = &logical.Request{
-			Operation: logical.CreateOperation,
-			Path:      staticRolePath + "hashicorp",
-			Storage:   storage,
-			Data:      data,
-		}
-
-		resp, err = b.HandleRequest(context.Background(), req)
+		resp, _ := createStaticRoleWithData(t, b, storage, "hashicorp", data)
```

Comment:
any reason not to handle the error here?

Reply:
It was being tested elsewhere, but I can handle it in the helper